meta/runtime.yml

```diff
@@ -1,2 +1,2 @@
 ---
-requires_ansible: '>=2.14.0'
+requires_ansible: '>=2.15.0'
```

Comment:
should get added by PRs from https://issues.redhat.com/browse/ACA-1753

Reply:
I shall wait for those PRs to be merged.